app/assets/javascripts/app/manage/statistics.js.coffee

```diff
@@ -0,0 +1,28 @@
+$ ->
+  if isOnPage 'manage', 'statistics'
+    uri = window.location.href
+
+    if uri.indexOf("?") < 0
+      $('.starts-at-wrapper .btn.all').addClass("active")
+    else
+      $('.starts-at-wrapper .btn-group .btn').addClass("active")
```

Comment:
I guess we must remove the active from the `all` button in this case, otherwise both buttons will look like they are pressed.